backend/src/main/java/com/benine/backend/database/MySQLDatabase.java

```diff
@@ -107,6 +208,9 @@ public void addPreset(Preset preset) {
       statement = connection.createStatement();
       String sql = preset.createAddSqlQuery();
       statement.executeUpdate(sql);
+      for (String tag : preset.getTags()) {
```

Comment:
I just realized there is another problem with the tagpresets table. Because we now have simplepresets and presets, but the tagpresets table only reffers to the presets table. This would imply the simplepresets can't have tags which should not be the case.